plugin/ipythonPlugins/src/dist/ipython/ipython.js

```diff
@@ -235,26 +270,34 @@ define(function(require, exports, module) {
             } else {
               evaluation.jsonres = jsonres;
               var elem = $(document.createElement("div"));
-              var oa = new myPython.OutputArea(elem);
+	      var oa = (ipyVersion == '3') ?
+                (new myPython.OutputArea({events: {trigger: function(){}},
```

Comment:
This block here is redundant: it should only be done in the else branch of the following if.